solutions/src/main/java/ru/sberbank/school/task08/JacksonSerializableManager.java

```diff
@@ -0,0 +1,79 @@
+package ru.sberbank.school.task08;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.NonNull;
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import ru.sberbank.school.task08.state.MapState;
+import ru.sberbank.school.task08.state.Savable;
+import ru.sberbank.school.util.Solution;
+
+import java.io.*;
+import java.util.List;
+
+@Solution(8)
+public class JacksonSerializableManager extends SaveGameManager {
+
+    private ObjectMapper objectMapper;
+    private String separator = File.separator;
+    /**
+     * Класс-наследник должен иметь конструктор эквивалентный этому.
+     *
+     * @param filesDirectoryPath Путь до директории, в которой хранятся файлы сохранений
+     */
+    public JacksonSerializableManager(@NonNull String filesDirectoryPath) {
```

Comment:
&#39;CTOR_DEF&#39; should be separated from previous statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a426709139>*